src/status_im/common/enter_seed_phrase/view.cljs

```diff
@@ -96,12 +89,24 @@
                                                            "keyboardDidHide"
                                                            #(reset! keyboard-shown? false))
                      invalid-seed-phrase?    (reagent/atom false)
+                     input-ref               (reagent/atom nil)
+                     on-input-ref            (fn [ref]
+                                               (reset! input-ref ref))
+                     focus-input             (fn []
+                                               (let [ref @input-ref]
+                                                 (when ref
+                                                   (.focus ref))))
                      set-invalid-seed-phrase #(reset! invalid-seed-phrase? true)
                      seed-phrase             (reagent/atom "")
                      on-change-seed-phrase   (fn [new-phrase]
                                                (when @invalid-seed-phrase?
                                                  (reset! invalid-seed-phrase? false))
                                                (reset! seed-phrase new-phrase))
+                     prepare-seed-phrase     (fn [seed-phrase]
+                                               (-> seed-phrase
+                                                   security/safe-unmask-data
+                                                   clean-seed-phrase
+                                                   security/mask-data))
```

Comment:
I think this can be extracted to a utility function as it may be resued else where